### PR TITLE
initial import for commenting

### PR DESCRIPTION
**Pull request by nbassler**

### CODE REVIEW

File: pytripgui/controller/plot_cont.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """
+    def __init__(self, model, fig):
+        self.model = model
+        self.fig = fig
+
+    @pyqtSlot()
+    def update_plot(self):
+        """
+        Updating plot.
+        What has to be plotted could be defined in model/. ? And in a seperate file?
```
Comment (grzanka) [inline pytripgui/controller/plot_cont.py]: The data to be plotted (i.e. ctx cube should go into the model), colors, look&feel ("how it is plotted" should be defined here).

File: pytripgui/view/README_autogen.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+pyuic5 main_window.ui > main_window.py
```
Comment (grzanka) [inline pytripgui/view/README_autogen.txt]: Can we move it somewhere else, i.e. to issue discussion ?

File: pytripgui/view/main_window.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'main_window.ui'
+#
+# Created by: PyQt5 UI code generator 5.9.2
```
Comment (grzanka) [inline pytripgui/view/main_window.py]: I don't like commiting autogenerated files, for packaging this file can be automatically generated.
Developers can also do it on their own.

Comment (nbassler) [reply pytripgui/view/main_window.py]: I thought so too. Where to place generation scrpits?

File: pytripgui/view/README_autogen.txt (hunk 2 above)
Comment (nbassler) [reply pytripgui/view/README_autogen.txt]: should move into a script... :)

File: pytripgui/view/main_window.py (hunk 3 above)
Comment (grzanka) [reply pytripgui/view/main_window.py]: Best - outside pytripgui directory, if small in a main dir, elsewhere in some subdirectory.

**Pull request merged**